### PR TITLE
Problem: not clear when & how to connect

### PR DESCRIPTION
**Pull request by splix**
Solution: show user details and running status

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -46,5 +46,9 @@ fn main() {
     let client_addr =
         args.flag_client_address.parse::<SocketAddr>().expect("Expect to parse client address");
 
+    println!("Starting Emerald Connector - v{}",
+             VERSION.unwrap_or("unknown"));
+    println!("Listen for connection on http://{}", addr);
+
```
Comment (dulanov) [inline src/main.rs]: Better to use log instructions, println! always a bad idea in production code, only for debugging one two times probably

Comment (dulanov) [inline src/main.rs]:
```Rust
    if log_enabled!(LogLevel::Info) {
        info!("Connector is started on {}", server.address());
    }
```

Comment (dulanov) [inline src/main.rs]: https://doc.rust-lang.org/log/log/index.html

**Comment by dulanov** [comment]
@splix PR's name isn't correspond to our aggrements, firstly to write about given problem.

**Comment by dulanov** [comment]
FYI: in order to see log records in console, use `RUST_LOG=emerald,rpc cargo run -- <agrs>`.

File: src/main.rs (hunk 1 above)
Comment (dulanov) [inline src/main.rs]: Your logic also duplicate another one from 'lib.rs'.

Comment (splix) [inline src/main.rs]: Ah, then it should log it b default. Right now you run app and wait, wait, wait, until you figure out it's already running just doesn't notify user about it

**Comment by splix** [comment]
Regarding PR, I thought probem/solution is for code comments

**Comment by dulanov** [comment]
No, also for PRs

File: src/main.rs (hunk 1 above)
Comment (dulanov) [inline src/main.rs]: use `RUST_LOG=emerald ...` prefix, we will fix it

**Comment by splix** [comment]
check new version

**Comment by dulanov** [comment]
Looks great, i merge it...

**Pull request merged**